src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: Avenir--book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.searchForm {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchForm--small {
+  height: 42px;
+}
+
+.searchForm:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.container {
+  position: relative;
+  width: 100%;
+  height: 100%;
```

Comment:
I think it's easier to understand and easier to create all search bar with inner container because without this container input width is not 100%. 
I fixed all mistakes in code and committed again.